src/lightning/pytorch/utilities/grads.py

```diff
@@ -48,5 +48,4 @@ def grad_norm(module: Module, norm_type: Union[float, int, str], group_separator
     if norms:
         total_norm = torch.tensor(list(norms.values())).norm(norm_type)
         norms[f"grad_{norm_type}_norm_total"] = total_norm
-    norms = {k: round(v.item(), 4) for k, v in norms.items()}
```

Comment:
For future reference, this was done in the past because this value would be automatically included in the progress bar when `track_gad_norm` was set. So norms this small would clutter the limited space in the progress bar.

This is no longer an issue after https://github.com/Lightning-AI/lightning/pull/16745

Reply:
I see. I think a better solution would have been to round at the location of display, i.e. before printing to the terminal, instead of during data collection, but now it doesn't matter anymore anyway.